### DISCUSSION TITLE
Expose ObjectRemovalHelper ?

### DISCUSSION THREAD

**Discussion by jailln** [iTowns/itowns#1910] [Ideas]
`ObjectRemovalHelper` contains helper methods to completely dispose of 3D objects, their material and textures. I think we should expose it in the API since it is quite a common thing that user might want to do when working with 3D objects directly imported with threejs (e.g. gltfs, collada, etc.) and hence it may help them. 

As a user I definitely need it :)

WDYT ? @mgermerie

**Comment by mgermerie** [comment]
I don't see an inconvenient to expose it. There is just a point on which I need a clarification : only the `cleanup` method does not need a `Layer`, and the other methods are called by more "generic" methods (such as the `removeLayer` from the view, or `dispose`). As a user, would it be useful to export these methods (those with a `Layer` parameter) ?

**Reply by jailln** [reply]
Yep you're right, we should only export the cleanup method
